devops/ci/single_node/deploy/Jenkinsfile

```diff
@@ -0,0 +1,140 @@
+#!/usr/bin/env groovy
+
+node {
+    properties([
+        buildDiscarder(logRotator(artifactDaysToKeepStr: '', artifactNumToKeepStr: '', daysToKeepStr: '5', numToKeepStr: '5')),
+        parameters([
+            string(defaultValue: '', description: 'FQDN of VM to deploy on.', name: 'HOST_NAME', trim: true),
+            string(defaultValue: '', description: '''Build having following directory structure:
+                3rd_party/
+                cortx_iso/
+                iso/
+                python_deps/
+                README.txt
+                RELEASE.INFO
+                THIRD_PARTY_RELEASE.INFO''', name: 'BUILD_URL', trim: true),
+            password(description: 'root user password for the target node.', name: 'PASSWORD')
+        ])
+    ])
+
+    def remote = [:]
+    withEnv(["CORTX_RELEASE_REPO=${BUILD_URL}","SSHPASS=${PASSWORD}"]) {
```

Comment:
makes sense to think about `withCredentials`

Reply:
withCredentials has a bug, moreover PASSWORD is used through the `sshCommand`  plugin.
We might want to mask password though.